Automattic/jetpack
# Instagram Widget: link to user connection if user is not connected

In Site-only connection, when trying to use the Instagram widget, the link suggests that you need to complete your connection by authorizing your user. However, the link goes to the Jetpack dashboard, which means scrolling down to find the box suggesting that you can connect to your WordPress.com account.

#### Changes proposed in this Pull Request:

* Instagram Widget: link to user connection if user is not connected

#### Jetpack product discussion


1191179647901802-as-1200235046043914

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Connect a site on site-level only
* Disable block widgets by adding `add_filter( 'use_widgets_block_editor', '__return_false' );`
* Go to Appearance > Widgets and add the Instagram Widget.
* You should see the message `In order to use this widget you need complete your Jetpack connection by authorizing your user.`
* Make sure the link takes you directly to `jetpack#/connect-user` and that you see the screen to connect you user
